flowauth/backend/flowauth/models.py

```diff
@@ -567,12 +569,11 @@ def init_db(force: bool = False) -> None:
         db.drop_all()
     db.create_all()
     current_app.config["DB_IS_SET_UP"].set()
+    # Since this is a fresh db, set the revision inside alembic to HEAD
+    flask_migrate.stamp(revision="head")
```

Comment:
Will this be executed if the db already exists (from previous deployment)? (Or if not, does it matter?) Not sure I've fully got my head around how Alembic determines revisions.

Reply:
No, this _should_ be skipped if the db already exists - in that case, there should either be an existing revision number or we're starting from revision = None. In the case of a fresh db (if I've understood Alembic correctly), we want to set it's revision to `head`.